cmd/options.go

```diff
@@ -336,6 +339,7 @@ func (o *Options) LogOptions() {
 	}
 	log.Info().Msgf("✨ - local-cluster-tool: %s", o.clusterProvider.GetName())
 	log.Info().Msgf("✨ - cluster-name: %s", o.ClusterName)
+	log.Info().Msgf("✨ - kind-options: %s", o.KindOptions)
```

Comment:
I suggest putting this into an if statement so it only prints the value if the option is set.
```suggestion
	if o.KindOptions != "" {
		log.Info().Msgf("✨ - kind-options: %s", o.KindOptions)
	}
```